2226/2/print_table.rb

```diff
@@ -0,0 +1,21 @@
+# This class is used for printing table to the terminal
+class PrintTable
```

Comment:
class names are nouns generally. `TablePrinte`r sounds much better

Reply:
this is basically a class that implements level 1 logic but according to its title, the only thing it does is table printing...